src/output_data.py

```diff
@@ -300,15 +307,23 @@ def write_plant_metadata(
     ]
 
     if not skip_outputs:
+        # From monthly EIA data, we want only the EIA-only subplants -- these are the ones that got shaped
+        monthly_eia_to_shape = eia923_allocated[
+            (eia923_allocated["hourly_data_source"] == "eia")
+            & ~(eia923_allocated["fuel_consumed_mmbtu"].isna())
+        ]
+
         # identify the source
         cems["data_source"] = "CEMS"
         partial_cems_subplant["data_source"] = "EIA"
         partial_cems_plant["data_source"] = "EIA"
         shaped_eia_data["data_source"] = "EIA"
+        monthly_eia_to_shape["data_source"] = "EIA"
 
         # identify net generation method
         cems = cems.rename(columns={"gtn_method": "net_generation_method"})
         shaped_eia_data["net_generation_method"] = shaped_eia_data["profile_method"]
+        monthly_eia_to_shape["net_generation_method"] = "synthetic_plant"
```

Comment:
So my understanding is that the plant metadata file will now have metadata for both the synthetic plant and all of the individual plants that make up a synthetic plant? If so, I think there are two options here: 
1. we might want to change this to "see shaped plant ID" so that it is clear that this is not a method, but instead telling the user to reference the methods for the synthetic plant id listed in the `shaped_plant_id` column
2. We could instead just copy the method used for the associated shaped plant id here

Reply:
If we go with option 2, we ptobably need to merge in the `shaped_plant_id` column first so we can use it as a key

Reply:
Went with option 1